ci/windows_ci_steps.sh

```diff
@@ -70,9 +70,10 @@ bazel "${BAZEL_STARTUP_OPTIONS[@]}" build "${BAZEL_BUILD_OPTIONS[@]}" //source/e
 
 # Copy binary to delivery directory
 cp -f bazel-bin/source/exe/envoy-static.exe "${ENVOY_DELIVERY_DIR}/envoy.exe"
+cp -f bazel-bin/source/exe/envoy-static.pdb "${ENVOY_DELIVERY_DIR}/envoy.pdb"
 
```

Comment:
For anyone observing from the bleacher seats... various attempts to resolve this are noted in https://github.com/envoyproxy/envoy/pull/16043